src/plugins/intel_npu/src/plugin/include/plugin.hpp

```diff
@@ -52,9 +53,9 @@ class Plugin : public ov::IPlugin {
                                                      const ov::SoPtr<ov::IRemoteContext>& context,
                                                      const ov::AnyMap& properties) const override;
 
-    std::shared_ptr<ov::ICompiledModel> import_model(ov::Tensor& stream, const ov::AnyMap& properties) const override;
+    std::shared_ptr<ov::ICompiledModel> import_model(ov::Tensor& tensor, const ov::AnyMap& properties) const override;
```

Comment:
```suggestion
    std::shared_ptr<ov::ICompiledModel> import_model(const ov::Tensor& compiled_blob, const ov::AnyMap& properties) const override;
```
To give description what tensor holds?
Should be const tensor as read only the (it will be not possible to get non-const pointer from 2026.0)

@olpipi 
Is it defined in core interface? 
If possible it should be const if we want only read data